README.md

```diff
@@ -20,6 +20,8 @@ go install ./...
 
 See [requirements](#requirements) for the list of libraries necessary for compilation.
 
+All significant (e.g. breaking) changes are documented in the [CHANGELOG.md](CHANGELOG.md).
```

Comment:
I would remove `(e.g. breaking)` because that makes it sound like there's a lot of breaking changes, whereas I'm not even sure when was the last breaking change.

Also, `documented in [CHANGELOG.md](CHANGELOG.md)` (no "the").